jsk_aero_robot/aeroeus/package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<package>
+  <name>aeroeus</name>
+  <version>1.1.0</version>
+  <description>The aeroeus package</description>
+
+  <license>BSD</license>
+
+  <maintainer email="youhei@jsk.imi.i.u-tokyo.ac.jp">
+    Yohei Kakiuchi
+  </maintainer>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <build_depend>pr2eus</build_depend>
+  <build_depend>aero_description</build_depend>
```

Comment:
if you want to `aero_description` here, please add them rosdistro file first. I you do not have plan for that, please remove or comment out this line

Reply:
removed aero_description